src/libawkward/array/ListArray.cpp

```diff
@@ -1725,7 +1725,7 @@ namespace awkward {
 
     Index64 missing = slicecontent.index();
     int64_t numvalid;
-    struct Error err1 = awkward_listarray_getitem_jagged_numvalid_64(
+    struct Error err1 = kernel::listarray_getitem_jagged_numvalid<int64_t>(
```

Comment:
This one was missed: it should be untemplated and the name should be `kernel::listarray_getitem_jagged_numvalid_64`.